src/composables/useLocaleRoute.ts

```diff
@@ -0,0 +1,58 @@
+import { useI18n } from "vue-i18n";
+
+export default function useLocaleRoute() {
+  const { t, locale: currentLocale, messages } = useI18n();
+
+  function localSegment(locale: string, segment: string) {
+    return messages.value[locale]["routes." + segment];
+  }
+
+  function genericRoute(locale: string, name: string) {
+    return `/${locale}/${name ? localSegment(locale, name) + "/" : ""}`;
+  }
+
+  const rootRoute = computed(() => `/${currentLocale.value}/`);
+  const helpRoute = computed(
+    () => `/${currentLocale.value}/${t("routes.help")}/`
+  );
+  const feedbackRoute = computed(
+    () => `/${currentLocale.value}/${t("routes.feedback")}/`
+  );
+  const buildingsRoute = computed(
+    () => `/${currentLocale.value}/${t("routes.buildings")}/`
+  );
+  function buildingRoute({
+    buildingSlug,
+    locale,
+  }: {
+    buildingSlug: string;
+    locale?: string;
+  }) {
+    const l = locale ?? currentLocale.value;
```

Comment:
move to helper function since it's used more than once?

```ts
const routeLocale = locale => locale ?? currentLocale.value;
// ...
return `${genericRoute(routeLocale(locale), "buildings")}${buildingSlug}/`;
```

Reply:
No more relevant since localised routes are now handled in a more generic way